src/components/ProductForm/ProductForm.js

```diff
@@ -83,7 +90,7 @@ function ProductForm({ onAddProduct, onAddCategories }) {
           Image Url
         </ScreenReaderOnlyLabel>
         <Input
-          type="text"
+          type="url"
           name="imgUrl"
           id="imgUrl"
           placeholder="Enter img url.."
```

Comment:
I'd use three dots or the ellipsis character instead of two dots :)
```suggestion
          placeholder="Enter img url…"
```